lld/test/MachO/cgdata-generate-merge.s

```diff
@@ -0,0 +1,85 @@
+# UNSUPPORTED: system-windows
+# REQUIRES: aarch64
+
+# RUN: rm -rf %t; split-file %s %t
```

Comment:
drive-by: you can add `cd %t` to remove `%t/` below

Reply:
I actually like keeping the `%t/` in the commands because it allows me to copy and run the commands directly for debugging without needing to first `cd %t`.

Reply:
Yes, I'll leave it as is, since each command is self-contained and complete, making it convenient to reproduce.